### PR TITLE
New/si/resultscss

### PR DESCRIPTION
**Pull request by siqbal181**

### CODE REVIEW

File: frontend/src/components/result-page/ResultPage.js [hunk 1]
```diff
@@ -67,19 +68,20 @@ const ResultPage = ({ navigate }) => {
   return (
     <>
       <div>
-        <button className="results-page-home">Home</button>
+        <button className="results-page-home">
+          <img src={HomeIcon} alt="home" onClick={() => navigate("/")} className="home-icon" />
+        </button>
```
Comment (umutbaykan) [inline frontend/src/components/result-page/ResultPage.js:73]: this needs to be another component as we will probably use it in the display story so far page. it also needs to reset all the values in the local storage when pressed, so the api calls do not carry over leftover data from previous calls

File: frontend/src/components/form-container/FormContainer.js [hunk 2]
```diff
@@ -28,7 +28,7 @@ const FormContainer = ({ navigate }) => {
   });
 
   useEffect(() => {
-    const animationDuration = 3000;
+    const animationDuration = 1;
```
Comment (umutbaykan) [inline frontend/src/components/form-container/FormContainer.js:31]: dont forget to reset this to 3000 prior to push :)

**Review by umutbaykan — CHANGES_REQUESTED** [review]

File: frontend/src/components/form-container/FormContainer.js (hunk 2 above)
Comment (siqbal181) [reply frontend/src/components/form-container/FormContainer.js]: ty! done

File: frontend/src/components/result-page/ResultPage.js (hunk 1 above)
Comment (siqbal181) [reply frontend/src/components/result-page/ResultPage.js]: I've removed this now as Speaking to @natashabuckham she is going to create the home component for local storage capabilty too. So its a static icon

**Review by umutbaykan — DISMISSED** [review]

**Comment by siqbal181** [comment]
merged into main and need another check pls @umutbaykan @natashabuckham

**Review by natashabuckham — APPROVED** [review]

**Pull request merged**